src/processor_x86.cpp

```diff
@@ -553,6 +556,10 @@ static CPU get_amd_processor_name(uint32_t family, uint32_t model, const uint32_
         if (model >= 0x30)
             return CPU::amd_znver2;
         return CPU::amd_znver1;
+    case 0x19:  // AMD Family 19h
+        if (model <= 0x0f || model == 0x21)
+            return CPU::amd_znver3;  // 00h-0Fh, 21h: Zen3
+        return CPU::generic;
```

Comment:
Should we return `amd_znver3` unconditionally as a baseline for Fam 19h?

Reply:
If you are telling me to do that (?) i can do that since i see that to be the usual pattern here,
but i've explicitly avoided doing that here because that is not what the other 4 sources of truth do:
* https://github.com/halide/Halide/blob/76793b48af0b29fd9958a63b97447a42c6a6654d/src/Target.cpp#L108-L149
* https://github.com/llvm/llvm-project/blob/955cff803e081640e149fed0742f57ae1b84db7d/llvm/lib/Support/Host.cpp#L968-L1041
* https://github.com/llvm/llvm-project/blob/955cff803e081640e149fed0742f57ae1b84db7d/compiler-rt/lib/builtins/cpu_model.c#L520-L586
* https://github.com/gcc-mirror/gcc/blob/000c1b89d259fadb466e1f2e63c79da45fd17372/gcc/common/config/i386/cpuinfo.h#L111-L264

Reply:
https://github.com/llvm/llvm-project/blob/955cff803e081640e149fed0742f57ae1b84db7d/compiler-rt/lib/builtins/cpu_model.c#L579-L586 this does set the CPU familiy to znver3, but not the subtype.
similar https://github.com/gcc-mirror/gcc/blob/000c1b89d259fadb466e1f2e63c79da45fd17372/gcc/common/config/i386/cpuinfo.h#L257-L263 

I would rather gracefully degrade to something that is an almost right schedule instead all the way to generic.